### ISSUE TITLE
not record to a file

### ISSUE THREAD

**Issue by diwu86** [goodatlas/react-native-audio-record#5]
Hi this is not an issue. I am wondering whether I can only listen to the mic but not save it to a file.  Basically I want to use 
```
AudioRecord.on('data', data => {
  // base64-encoded audio data chunks
}); 
 ```
without save the file to my phone. Is it possible? Thanks

**Comment by vinayr** [comment]
@diwu86 this is not possible at the moment. You can just ignore the file. Does saving audio to file cause any issue?

**Comment by diwu86** [comment]
No. Saving file is working fine. I have to stop and re-start cause I don't want to make the file too big.

**Comment by brycepavey** [comment]
@diwu86 how are you playing/storing the data from the chunks?

**Comment by kolnogorov** [comment]
And how can i delete the file after the recording?

**Comment by maxckelly** [comment]
Was wondering if anyone has been able to do this? We're uploading chunks to our server and processing into a file there. We have experienced a number of crashes on two devices with 32gb and 23gb available. One crashing at around 90mins or recording and another around 70mins.

Package is great though thank you for your hard work!

**Comment by brycepavey** [comment]
@maxckelly a colleague of mine implemented a circular buffer forked from this repo for an app we developed for this purpose. 
It was a while ago so I can’t remember exactly the specifics of the implementation, you’ll have to look through the commit history to figure it out as we didn’t document any of it but hopefully it’s enough to get you going. 
https://github.com/dreamwalk-os/react-native-audio-record

**Comment by jecs89** [comment]
I could get not record to a file, commenting these lines in java file: RNAudioRecordModule.java
//saveAsWav();
//stopRecordingPromise.resolve(outFile);
